src/organic-keywords/handler.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import {
+  internalServerError, noContent, notFound, ok,
+} from '@adobe/spacecat-shared-http-utils';
+import { retrieveSiteBySiteId } from '../utils/data-access.js';
+import AhrefsAPIClient from '../support/ahrefs-client.js';
+
+export default async function auditOrganicKeywords(message, context) {
+  const { type, url: siteId, auditContext } = message;
+  const { dataAccess, log, sqs } = context;
+  const {
+    AUDIT_RESULTS_QUEUE_URL: queueUrl,
+  } = context.env;
+
+  try {
+    log.info(`Received ${type} audit request for siteId: ${siteId}`);
+
+    const site = await retrieveSiteBySiteId(dataAccess, siteId, log);
+
+    if (!site) {
+      return notFound('Site not found');
+    }
+
+    const auditConfig = site.getAuditConfig();
+    if (auditConfig.auditsDisabled()) {
+      log.info(`Audits disabled for site ${siteId}`);
+      return ok();
+    }
+
+    if (auditConfig.getAuditTypeConfig(type)?.disabled()) {
+      log.info(`Audit type ${type} disabled for site ${siteId}`);
+      return ok();
+    }
+
+    const ahrefsAPIClient = AhrefsAPIClient.createFrom(context);
+    const response = await ahrefsAPIClient.getOrganicKeywords(site.getBaseURL(), auditContext);
```

Comment:
How do you query and get results from ahrefs, using the www or non-www version of the site URL?

Reply:
shouldn't it be exactly the same as the saved base url? I tested in ahrefs and if it's not the same it does not return the right results

Reply:
I've checked ahrefs UI with `bamboohr.com` and it returns no results, whereas www.bamboohr.com does.
![Screenshot 2024-01-30 at 14 51 32](https://github.com/adobe/spacecat-audit-worker/assets/5692662/4cd8b04d-d91c-4375-bd99-c426ce831f20)
![Screenshot 2024-01-30 at 14 51 04](https://github.com/adobe/spacecat-audit-worker/assets/5692662/91da701d-f44f-4a47-9947-d5dc05b49092)

Reply:
`bamboohr.com` also forwards to `www.bamboohr.com`. Shouldn't we have `www.bamboohr.com` as a baseURL in that case?